kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerTransporter.kt

```diff
@@ -153,7 +153,7 @@ internal class DataBrokerTransporter(
                     val entry = entryUpdate.entry
 
                     subscription.listeners.forEach { observer ->
```

Comment:
Let's discuss this part. It may not be necessary anymore to loop through every entry here (splitting them) and then fusing them together again in the PropertyListener here:

```
runBlocking(specificationUpdateContext) {
    updatedVssSpecification = updatedVssSpecification.copy(vssPath, updatedValue.value)
}
```

We could remove the blocking and the caching via the updatedVssSpecification property if we work with the whole response list at once. The initialSubscriptionUpdates are probably also not necessary anymore.

Reply:
Yeah, let's discuss it once you're back